oauth2_provider/migrations/0005_accesstoken_id_token.py

```diff
@@ -0,0 +1,20 @@
+from django.conf import settings
```

Comment:
would be nice if you could remove extra migrations files and regen them

Reply:
Done...